Resideo_eshopping/resideo_eshopping/lib/Screens/screen3.dart

```diff
@@ -0,0 +1,185 @@
+import 'package:flutter/material.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+
+class Screen3 extends StatelessWidget {
```

Comment:
Please rename the class name which describes it's functionality.

Reply:
done